### PR TITLE
fix: removed p2p related videos

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [x] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by NikitK-deriv — DISMISSED** [review]

File: src/pages/academy/index.tsx [hunk 1]
```diff
@@ -91,7 +93,9 @@ const DerivBlog = ({ data }: DerivBlogProps) => {
     let recent_data = data.directus.recent
     let featured_data = data.directus.featured
     let homepage_banner_data = data.directus.homepage_banners
-    let non_featured_video_list_data = data.directus.videos
+    let non_featured_video_list_data = data.directus.videos.filter((item) => {
+        return !is_p2p_allowed_country ? !item.video_title.includes('Deriv P2P') : item
```
Comment (habib-deriv) [inline src/pages/academy/index.tsx:97]: Let's get this text 'Deriv P2P' from a constant

File: src/store/index.tsx [hunk 2]
```diff
@@ -29,17 +29,32 @@ export const DerivStore = createContext<DerivStoreType>(null)
...
     const [user_country, setUserCountry] = useState(null)
-
     const breakpoints = useBreakpoints()
 
+    const useAvailabileP2pData = () => {
+        let [filtered_academy_video] = useAcademyData()
+        let [filtered_academy_blogs] = useAcademyData()
+
+        filtered_academy_video = filtered_academy_video.videos.filter((item) => {
+            return !is_p2p_allowed_country ? !item.video_title.includes('Deriv P2P') : item
```
Comment (habib-deriv) [inline src/store/index.tsx:45]: Same here

File: src/store/index.tsx [hunk 3]
```diff
@@ -29,17 +29,32 @@ export const DerivStore = createContext<DerivStoreType>(null)
...
+        })
+
+        filtered_academy_blogs = filtered_academy_blogs.blog.filter((item) => {
+            return !is_p2p_allowed_country ? !item.blog_title.includes('Deriv P2P') : item
```
Comment (habib-deriv) [inline src/store/index.tsx:49]: Same here

File: src/pages/academy/index.tsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/pages/academy/index.tsx:97]: okay

File: src/store/index.tsx (hunk 2 above)
Comment (aswathy-deriv) [reply src/store/index.tsx:45]: okay

File: src/store/index.tsx (hunk 3 above)
Comment (aswathy-deriv) [reply src/store/index.tsx:49]: okay

File: src/pages/help-centre/index.tsx [hunk 4]
```diff
@@ -9,11 +9,15 @@ import { SEO, Desktop, Container } from 'components/containers'
...
 import { usePlatformQueryParam } from 'components/hooks/use-platform-query-param'
+import { DerivStore } from 'store'
 
 const HelpCentre = () => {
     const { is_deriv_go } = usePlatformQueryParam()
+    const { is_p2p_allowed_country } = React.useContext(DerivStore)
     const general_questions = getQuestionsBySection(GENERAL)
-    const platforms_questions = getQuestionsBySection(PLATFORMS)
+    const platforms_questions = getQuestionsBySection(PLATFORMS).filter((item) => {
+        return !is_p2p_allowed_country ? !item.category.includes('_t_Deriv P2P_t_') : item
```
Comment (habib-deriv) [inline src/pages/help-centre/index.tsx:19]: Hi @aswathy-deriv , in the include, do we need to have the "_t_Deriv P2P_t_" translation key?

Comment (aswathy-deriv) [reply src/pages/help-centre/index.tsx]: okay i removed it

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Review by NikitK-deriv — DISMISSED** [review]

**Review by prince-deriv — APPROVED** [review]

**Pull request closed without merging**